upload.py

```
@@ -91,95 +111,79 @@ def gemini_generate(prompt, task_name):
+        raise ValueError("GITHUB_TOKEN not set")
+
+    headers = {"Authorization": f"token {token}"}
+    url_create_pr = f"https://api.github.com/repos/{repo}/pulls"
+    payload = {
+        "title": pr_title,
+        "head": branch_name,
+        "base": base_branch,
+        "body": f"Commit Message:\n{commit_message}\n\nGenerated automatically.",
+    }
+    if not args.dry_run:
+        response = requests.post(url_create_pr, json=payload, headers=headers)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)